nih-sparc/sparc-app
# Organ Detail Page Initial Setup #4a74zu

# Description

The purpose of this PR is to get the initial Organ detail page setup. Note that data is not yet hooked up to contentful.

## Type of change


# How Has This Been Tested?

1. Navigate to the Find Data page and go to the Organs tab
2. Click on the Organ listed in the table. The Organ details page should load and be displayed as expected.

# Checklist:
